CNAME

```diff
@@ -1 +0,0 @@
-errorprone.info
```

Comment:
Thanks for the catch. I don't want to merge this, it was just for testing with my local fork. (Otherwise github complains because two repositories are trying to use the same CNAME...)

I'll get rid of it before merging.